src/chains/commonwealth/subscribeFunc.ts

```diff
@@ -50,7 +50,7 @@ export async function createApi(
 
       // create subscriptions for all existing projects
       const nProjects = await api.factory.numProjects();
-      for (let projectN = 1; projectN <= nProjects.toNumber(); projectN++) {
+      for (let projectN = 1; projectN <= nProjects; projectN++) {
```

Comment:
Why remove the toNumber?

Reply:
removed because numProjects is not a string anymore but a number in the Contract

Reply:
Numbers in contracts are typically returned as BigNumber so we store them as strings, unless we can verify that a JS number has sufficient precision to represent it.